app/main.py

```diff
@@ -1,8 +1,56 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int = 1,
+            clean_mark: int = 1,
+            brand: str = ""
+    ) -> None:
+
+        self.comfort_class = comfort_class if 1 <= comfort_class <= 7 else 1
+        self.clean_mark = clean_mark if 1 <= clean_mark <= 10 else 1
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float = 1.0,
+            clean_power: int = 1,
+            average_rating: float = 1.0,
+            count_of_ratings: int = 0
+    ) -> None:
+
+        self.distance_from_city_center = distance_from_city_center \
```

Comment:
As per our style guidelines we don't use `\` for line breakers, here you can use `()`  instead